### PR TITLE
Make deploy-webapi do all it needs to by default

### PR DESCRIPTION
**Pull request by glahaye**
### Motivation and Context
Some switches need to be included at the command line to ensure proper modification of the AAD app registration and addition of our URIs to the plugin CORS. These steps should be done by default.

### Description
Make all required steps run by default.
Fixed population of frontend version info

### Contribution Checklist
- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows the [Contribution Guidelines](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [ ] All unit tests pass, and I have added new tests where possible

### CODE REVIEW

File: scripts/deploy/deploy-webapi.ps1 [hunk 1]
```diff
@@ -27,12 +27,12 @@ param(
     $PackageFilePath = "$PSScriptRoot/out/webapi.zip",
 
     [switch]
-    # Switch to add our URIs in app registration's redirect URIs if missing
-    $EnsureUriInAppRegistration,
+    # Don't attempt to add our URIs in app registration's redirect URIs
```
Comment (glahaye) [inline scripts/deploy/deploy-webapi.ps1:30]: Yes, this changes the existing logic.  But given this the current logic has only worked from this week and that people are already complaining about how it is incomplete, I'm fixing it now to kill this problem right away.

Comment (TaoChenOSU) [reply scripts/deploy/deploy-webapi.ps1:30]: nit: maybe worth mentioning it's the WebApp's or the frontend's end registration.

File: .github/workflows/copilot-deploy-backend.yml [hunk 2]
```diff
@@ -81,4 +81,4 @@ jobs:
 
       - name: "Deploy"
         run: |
-          scripts/deploy/deploy-webapi.sh -p "${{ github.workspace }}/${{inputs.ARTIFACT_NAME}}/webapi.zip" -d ${{inputs.DEPLOYMENT_NAME}} -s ${{secrets.AZURE_SUBSCRIPTION_ID}} -rg ${{vars.CC_DEPLOYMENT_GROUP_NAME}} --register-cors
+          scripts/deploy/deploy-webapi.sh -p "${{ github.workspace }}/${{inputs.ARTIFACT_NAME}}/webapi.zip" -d ${{inputs.DEPLOYMENT_NAME}} -s ${{secrets.AZURE_SUBSCRIPTION_ID}} -rg ${{vars.CC_DEPLOYMENT_GROUP_NAME}} --skip-app-registration
```
Comment (TaoChenOSU) [inline .github/workflows/copilot-deploy-backend.yml:84]: If we do this, will the app get redirected back correctly after authentication?

File: scripts/deploy/deploy-webapi.ps1 (hunk 1 above)
Comment (glahaye) [reply scripts/deploy/deploy-webapi.ps1]: Done

**Review by TaoChenOSU — APPROVED** [review]

File: scripts/deploy/deploy-webapi.ps1 (hunk 1 above)
Comment (glahaye) [reply scripts/deploy/deploy-webapi.ps1]: Done

File: .github/workflows/copilot-deploy-backend.yml (hunk 2 above)
Comment (glahaye) [reply .github/workflows/copilot-deploy-backend.yml:84]: No change from previous behavior. This actually makes the absence of registration more obvious

**Pull request merged**